examples/howto/plot_batch_simulate.py

```diff
@@ -95,36 +107,58 @@ def summary_func(results):
 ###############################################################################
 # Run the batch simulation and collect the results.
 
-# Comment off this code, if dask and distributed Python packages are installed
+# Uncomment this code if dask and distributed Python packages are installed.
 # from dask.distributed import Client
 # client = Client(threads_per_worker=1, n_workers=5, processes=False)
```

Comment:
Concerning all Dask lines (in the later version, Lines 110-112 and 124 of `plot_batch_simulate.py`): After installing the packages, uncommenting these lines, and changing the `backend`, I could not get this to work at all. On both Macos on Apple Silicon and Linux on Intel chips via Oscar, I could not get the `dask` usage case to work. I tested different numbers of workers and some options, but in every case I either got NEURON errors about actual simulations or just segmentation faults. Also, we have no tests for the `dask` cases at all. @ntolley @samadpls Do y'all use, or are you aware of anyone who uses, the `dask` backends for batch simulation, or at all?

I think that, for now, we should remove any mention of `dask` from this Example. If we want to truly support `dask`, then there is a good amount of work that needs to be done (illustrating that it works, writing tests for it including across platforms, justification to use it in place of JobLib, etc.).